lib/reek/spec/should_reek_of.rb

```diff
@@ -8,6 +8,7 @@ module Spec
     # An rspec matcher that matches when the +actual+ has the specified
     # code smell.
     #
+    # :reek:InstanceVariableAssumption
```

Comment:
When I look at the code of this class, I think this class does not have this smell.